<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlatformEdit</class>
 <widget class="QWidget" name="PlatformEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>498</width>
    <height>303</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>498</width>
    <height>303</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>498</width>
    <height>303</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Platform Editor</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>461</width>
     <height>243</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>6</number>
    </property>
    <property name="topMargin">
     <number>6</number>
    </property>
    <property name="rightMargin">
     <number>6</number>
    </property>
    <property name="bottomMargin">
     <number>6</number>
    </property>
    <property name="horizontalSpacing">
     <number>10</number>
    </property>
    <property name="verticalSpacing">
     <number>5</number>
    </property>
    <item row="0" column="3">
     <widget class="QCheckBox" name="plat_name_check">
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="6" column="3">
     <widget class="QCheckBox" name="share_deal_fee_check">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="FastEditQDoubleSpinBox" name="share_plat_max_fee_box">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::ButtonSymbols::NoButtons</enum>
      </property>
      <property name="correctionMode">
       <enum>QAbstractSpinBox::CorrectionMode::CorrectToNearestValue</enum>
      </property>
      <property name="prefix">
       <string>£</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QCheckBox" name="fund_deal_fee_check">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="FastEditQDoubleSpinBox" name="share_deal_fee_box">
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::ButtonSymbols::NoButtons</enum>
      </property>
      <property name="correctionMode">
       <enum>QAbstractSpinBox::CorrectionMode::CorrectToNearestValue</enum>
      </property>
      <property name="prefix">
       <string>£</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0" colspan="2">
     <widget class="QLabel" name="share_deal_reduce_trades_lab">
      <property name="text">
       <string>Share dealing discount # of trades</string>
      </property>
     </widget>
    </item>
    <item row="9" column="3">
     <widget class="QCheckBox" name="share_deal_reduce_amount_check"/>
    </item>
    <item row="5" column="0" colspan="2">
     <widget class="QLabel" name="share_plat_max_fee_lab">
      <property name="text">
       <string>Share platform monthly fee cap</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="FastEditQDoubleSpinBox" name="fund_deal_fee_box">
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::ButtonSymbols::NoButtons</enum>
      </property>
      <property name="correctionMode">
       <enum>QAbstractSpinBox::CorrectionMode::CorrectToNearestValue</enum>
      </property>
      <property name="prefix">
       <string>£</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="2">
     <widget class="QLabel" name="share_deal_fee_lab">
      <property name="text">
       <string>Share dealing fee*</string>
      </property>
     </widget>
    </item>
    <item row="8" column="2">
     <widget class="FastEditQSpinBox" name="share_deal_reduce_trades_box">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::ButtonSymbols::NoButtons</enum>
      </property>
      <property name="correctionMode">
       <enum>QAbstractSpinBox::CorrectionMode::CorrectToNearestValue</enum>
      </property>
     </widget>
    </item>
    <item row="8" column="3">
     <widget class="QCheckBox" name="share_deal_reduce_trades_check"/>
    </item>
    <item row="4" column="3">
     <widget class="QCheckBox" name="share_plat_fee_check">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QLabel" name="fund_deal_fee_lab">
      <property name="text">
       <string>Fund dealing fee*</string>
      </property>
     </widget>
    </item>
    <item row="5" column="3">
     <widget class="QCheckBox" name="share_plat_max_fee_check"/>
    </item>
    <item row="0" column="2">
     <widget class="QLineEdit" name="plat_name_box"/>
    </item>
    <item row="9" column="0" colspan="2">
     <widget class="QLabel" name="share_deal_reduce_amount_lab">
      <property name="text">
       <string>Share dealing discount amount</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="2">
     <widget class="QLabel" name="share_plat_fee_lab">
      <property name="text">
       <string>Share platform fee*</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="FastEditQDoubleSpinBox" name="share_plat_fee_box">
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::ButtonSymbols::NoButtons</enum>
      </property>
      <property name="correctionMode">
       <enum>QAbstractSpinBox::CorrectionMode::CorrectToNearestValue</enum>
      </property>
      <property name="suffix">
       <string>%</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="plat_name_lab">
      <property name="text">
       <string>Platform name</string>
      </property>
     </widget>
    </item>
    <item row="9" column="2">
     <widget class="FastEditQDoubleSpinBox" name="share_deal_reduce_amount_box">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::ButtonSymbols::NoButtons</enum>
      </property>
      <property name="correctionMode">
       <enum>QAbstractSpinBox::CorrectionMode::CorrectToNearestValue</enum>
      </property>
      <property name="prefix">
       <string>£</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="save_but">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>270</y>
     <width>141</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QLabel" name="req_field_lab">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>* Indicates required field</string>
   </property>
  </widget>
  <widget class="QLabel" name="active_lab">
   <property name="geometry">
    <rect>
     <x>437</x>
     <y>10</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Active?</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FastEditQDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>widgets/fastedit_spinbox</header>
  </customwidget>
  <customwidget>
   <class>FastEditQSpinBox</class>
   <extends>QSpinBox</extends>
   <header>widgets/fastedit_spinbox</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>plat_name_box</tabstop>
  <tabstop>fund_deal_fee_box</tabstop>
  <tabstop>share_plat_fee_box</tabstop>
  <tabstop>share_plat_max_fee_box</tabstop>
  <tabstop>share_deal_fee_box</tabstop>
  <tabstop>share_deal_reduce_trades_box</tabstop>
  <tabstop>share_deal_reduce_amount_box</tabstop>
  <tabstop>save_but</tabstop>
  <tabstop>plat_name_check</tabstop>
  <tabstop>fund_deal_fee_check</tabstop>
  <tabstop>share_plat_fee_check</tabstop>
  <tabstop>share_plat_max_fee_check</tabstop>
  <tabstop>share_deal_fee_check</tabstop>
  <tabstop>share_deal_reduce_trades_check</tabstop>
  <tabstop>share_deal_reduce_amount_check</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>save_but</sender>
   <signal>clicked()</signal>
   <receiver>PlatformEdit</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>445</x>
     <y>211</y>
    </hint>
    <hint type="destinationlabel">
     <x>275</x>
     <y>117</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
